lib/dispatch-rider/logging/translator/base_translator.rb

```diff
@@ -0,0 +1,40 @@
+module DispatchRider
+  module Logging
+    class Translator
+
+      class BaseTranslator
```

Comment:
Since you're calling this `Base` you should define the common interface. The next dev to create a `Translator` would then just copy the interface and implement them.


```ruby
def translate
  raise NotImplementedError, "Translators must implement #translate."
end
```